sts_synindex_external.R

```diff
@@ -60,16 +61,17 @@ duplicate_folder(source_folder = PARQUET_FILTERED_LOCATION,
                  destination_folder = PARQUET_FINAL_LOCATION)
 
 copy_folders_reparent <- function(source_folder, destination_folder) {
```

Comment:
Nit: can we follow roxygen2 documentation standards? https://r-pkgs.org/man.html#roxygen2-basics

Reply:
See commit [50829db](https://github.com/Sage-Bionetworks/recover-parquet-external/pull/4/commits/50829dbd454e7e812aec5906d08b52f99f67068f)

Reply:
I thought roxygen documentation was only to be used for in R packages, so I omitted a few of the roxygen tags like `examples` and `export`. Are the changes I pushed in that commit in line with what you meant by following roxygen2 documentation standards? @thomasyu888

Reply:
Ya, you can probably remove the `export` tags, but the `examples` is helpful still.

Reply:
[34eaeb8](https://github.com/Sage-Bionetworks/recover-parquet-external/pull/4/commits/34eaeb88f59caed607bc5883dd8014e06bed8799)